### DISCUSSION TITLE
[Pull Shark Achievement] I have multiple pull requests to open-source projects but why there's only one showing up?

### DISCUSSION THREAD

**Discussion by mavyfaby** [community/community#29603] [Other Feature Feedback, Questions, & Ideas] [answered]
As the title proposes, there's only one repo showing up in my `Pull shark` [achievement](https://github.com/users/mavyfaby/achievements/pull-shark). 

![image](https://user-images.githubusercontent.com/51808724/184782757-e07bddc4-b828-41df-a414-6d3e88af1331.png)

![image](https://user-images.githubusercontent.com/51808724/184782679-2e64129b-c984-424e-ae7e-707565014b1e.png)

I'm not sure if it's a bug or it only shows 1 repo for some reason.

**Comment by radar-blip** [comment]
Thats how it is supposed to be, main thing is you merged 2 pull requests. Its fine.
It doesn't matter if its the same repo, or not.

**Comment by lbirkert** [comment, marked as answer]
> As the title proposes, there's only one repo showing up in my `Pull shark` [achievement](https://github.com/users/mavyfaby/achievements/pull-shark).
> 
> ![image](https://user-images.githubusercontent.com/51808724/184782757-e07bddc4-b828-41df-a414-6d3e88af1331.png)
> 
> ![image](https://user-images.githubusercontent.com/51808724/184782679-2e64129b-c984-424e-ae7e-707565014b1e.png)
> 
> I'm not sure if it's a bug or it only shows 1 repo for some reason.

You unlocked the achievement by merging your second pull request. That's the pull request that shows up. You can get level 2 of pull shark by merging your 16th pull request.
